src/json/mapData.json

```diff
@@ -0,0 +1,2427 @@
+{
```

Comment:
This is _way_ too big. Should this even be in the repository, or should we be getting it from a data store?